app/templates/edit_section.html

```diff
@@ -3,9 +3,9 @@
 
 {% extends "_base_page.html" %}
 
-{% set page_title = "Edit {}".format(section.name) %}
+{% set pageTitle = "Edit {}".format(section.name) %}
```

Comment:
This breaks the breadcrumbs, which is looking for `page_title`

Reply:
For the other apps, will it be possible to split out the pageTitle updates into their own tickets so we have greater visibility of this? There are a lot of changes in this PR to check...

Reply:
we could it just means more temporary workarounds to add before taking them out at a later stage. I do understand your point and I think going forward we should be able to break this out into another PR